Bearded.UI/Controls/CompositeControl.cs

```diff
@@ -46,7 +47,13 @@ public void RemoveAllChildren()
             }
         }
 
-        public bool FocusDescendant(Control control) => Parent.FocusDescendant(control);
+        public bool FocusDescendant(Control control)
+        {
+            if (Parent == null)
```

Comment:
nit: I prefer always using brackets, even for single-line ifs.

Reply:
I agree, but not using brackets is consistent with the styling used in the rest of the repo.

I can use brackets wherever I've added code, but that would be almost the only places (for single lines).

Reply:
If this is consistent with the rest of the code, it's fine to keep. We should then probably do a mass change to make this more consistent.